models/UserModel.ts

```diff
@@ -87,6 +87,22 @@ export class UserModel extends BaseEntity {
     return this.accessType === UserAccessType.STAFF;
   }
 
+  public hasStoreDistributorPermissions(): boolean {
```

Comment:
This should be `PermissionsService::canDistributeMerch` or something.

Reply:
^^^^ this